packages/brick_sqlite_generators/lib/src/sqlite_schema/migration_generator.dart

```diff
@@ -49,90 +50,105 @@ class MigrationGenerator extends Generator {
+              reader.read('name').stringValue,
+            );
+          } else if (_insertColumnChecker.isExactlyType(object.type!)) {
+            final definitionObject = reader.read('definitionType').objectValue;
+            final columnIndex = definitionObject.getField('index')!.toIntValue()!;
+            final definitionValue = Column.values[columnIndex];
+            return InsertColumn(
+              reader.read('name').stringValue,
+              definitionValue,
+              autoincrement: reader.read('autoincrement').isNull
+                  ? InsertColumn.defaults.autoincrement
+                  : reader.read('autoincrement').boolValue,
+              defaultValue: reader.read('defaultValue').isNull
+                  ? InsertColumn.defaults.defaultValue
+                  : reader.read('defaultValue').literalValue,
+              nullable: reader.read('nullable').isNull
+                  ? InsertColumn.defaults.nullable
+                  : reader.read('nullable').boolValue,
+              onTable: reader.read('onTable').stringValue,
+              unique: reader.read('unique').isNull
+                  ? InsertColumn.defaults.unique
+                  : reader.read('unique').boolValue,
+            );
+          } else if (_insertForeignKeyChecker.isExactlyType(object.type!)) {
+            return InsertForeignKey(
+              reader.read('localTableName').stringValue,
+              reader.read('foreignTableName').stringValue,
+              foreignKeyColumn: reader.read('foreignKeyColumn').isNull
+                  ? null
+                  : reader.read('foreignKeyColumn').stringValue,
+              onDeleteCascade: !reader.read('onDeleteCascade').isNull &&
+                  reader.read('onDeleteCascade').boolValue,
+              onDeleteSetDefault: !reader.read('onDeleteSetDefault').isNull &&
+                  reader.read('onDeleteSetDefault').boolValue,
+            );
+          } else if (_insertTableChecker.isExactlyType(object.type!)) {
+            return InsertTable(
+              reader.read('name').stringValue,
+            );
+          } else if (_renameColumnChecker.isExactlyType(object.type!)) {
+            return RenameColumn(
+              reader.read('oldName').stringValue,
+              reader.read('newName').stringValue,
+              onTable: reader.read('onTable').stringValue,
+            );
+          } else if (_renameTableChecker.isExactlyType(object.type!)) {
+            return RenameTable(
+              reader.read('oldName').stringValue,
+              reader.read('newName').stringValue,
+            );
+          } else if (_migrationCommandChecker.isAssignableFromType(object.type!)) {
+            return null;
+          } else {
+            throw UnimplementedError(
+              'Cannot create migration line for ${object.type}',
+            );
+          }
+        })
+        .where((command) => command != null)
+        .cast<MigrationCommand>()
```

Comment:
To simplify things. But it still feels like this should never be null?
```suggestion
        .whereType<MigrationCommand>()
```